scripts/test-api-entrypoint.sh

```
@@ -0,0 +1,131 @@
+trap 'rm -rf "$work"' EXIT
+failures=0
+
+# run PROBE_ANSWER PASSWORD [preexisting-file-contents]
+#
+# PROBE_ANSWER is what the stubbed `margince-migrate org-exists` reports:
+# "true", "false", or "fail" to exit non-zero without answering. Echoes the
+# entrypoint's exit status; leaves its output in $out and the credential file at
+# $pwfile for the caller to assert on.
+run() {
+    local probe="$1" password="${2:-}" preexisting="${3:-}"
+    rm -rf "$work/app" "$work/bin"
```

Shellcheck (0.11.0)

[warning] Use "${var:?}" to ensure this never expands to /bin .

(SC2115)